src/components/TreeImageScrubber.js

```diff
@@ -480,6 +480,7 @@ const TreeImageScrubber = (props) => {
               buttons={[
                 <IconButton onClick={handleFilterClick} key={1}>
```

Comment:
This would be better as a regular `Button` with a `startIcon`.
See https://material-ui.com/components/buttons/#buttons-with-icons-and-label.

Reply:
I have fixed those issues in my latest commit.

Reply:
Thanks @tranquanghuy0801.
I hope you don't mind, I've made a couple of tweaks to the styling to centre the button vertically and remove the outline.